docs/src/modules/java/pages/author-your-first-service.adoc

```diff
@@ -182,10 +185,120 @@ include::example$doc-snippets/src/main/java/com/example/api/ExampleEndpoint.java
 ----
 <1> Returning a record that gets serialized as JSON
 
+Restart the service and curl this command:
+
 [source, command line]
 ----
 curl localhost:9000/hello/hello/Bob/30
 ----
 
+== Explore the local console
+
+The Akka local console is a web-based tool that provides a convenient way to view and interact with your running service.
+
+=== Install the Akka CLI
+
+The local console requires that you install the Akka CLI first.
+
+Install the Akka CLI:
+
+[.tabset]
+Linux::
++
+--
+Download and install the latest version of `akka`:
+[source,bash]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash
+....
+
+If that fails due to permission issues, use:
+
+[source,bash]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- --prefix /tmp && \
+    sudo mv /tmp/akka /usr/local/bin/akka
+....
+
+You can pass options to the installer script with `-s --` e.g.:
+
+[source,bash,subs="attributes"]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- --prefix=$HOME --version={akka-cli-version} --verbose
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- -P $HOME -v {akka-cli-version} -V
```

Comment:
if we are anyway not linking or including it as a partial I think we can simplify the installation instructions here by just showing the normal commands, and not all options.

Reply:
and the normal command is without version, to install latest